src/daffodilDebugger/debugger.ts

```diff
@@ -57,22 +58,42 @@ async function getTDMLConfig(
     // Might need to add `schema` here if we move the `Execute TDML` command
     //   away from the detected dfdl language in VSCode.
     config.data = ''
+    config.schema.path = ''
 
     if (config?.tdmlConfig?.path === undefined)
       config.tdmlConfig.path = await vscode.commands.executeCommand(
         'extension.dfdl-debug.getValidatedTDMLPath'
       )
-  }
 
-  if (
-    config?.tdmlConfig?.action === 'generate' ||
-    config?.tdmlConfig?.action === 'execute'
-  ) {
     if (config?.tdmlConfig?.name === undefined)
-      config.tdmlConfig.name = getDefaultTDMLTestCaseName()
+      config.tdmlConfig.name = await vscode.commands.executeCommand(
+        'extension.dfdl-debug.getTDMLName'
+      )
 
     if (config?.tdmlConfig?.description === undefined)
+      config.tdmlConfig.description = await vscode.commands.executeCommand(
+        'extension.dfdl-debug.getTDMLDescription'
+      )
+  }
+
+  if (config?.tdmlConfig?.action === 'generate') {
+    if (
+      config?.tdmlConfig?.name === undefined ||
+      config?.tdmlConfig?.name === 'undefined'
```

Comment:
Concern: this looks really wonky. Why is it needed?

Reply:
This is definitely not needed, this is a harmless workaround to handle current launch config wizard implementation. We are planning to create another technical debt or bug issue #1024 is created to handle this.